server/datastore/mysql/mdm.go

```diff
@@ -1213,3 +1213,60 @@ func getTableAndColumnNameForHostMDMProfileUUID(profUUID string) (table, column
 		return "", "", fmt.Errorf("invalid profile UUID prefix %s", profUUID)
 	}
 }
+
+func (ds *Datastore) AreHostsConnectedToFleetMDM(ctx context.Context, hosts []*fleet.Host) (map[string]bool, error) {
+	var (
+		appleUUIDs []any
+		winUUIDs   []any
+	)
+
+	res := make(map[string]bool, len(hosts))
+	for _, h := range hosts {
+		switch h.Platform {
+		case "darwin", "ipados", "ios":
+			appleUUIDs = append(appleUUIDs, h.UUID)
+		case "windows":
+			winUUIDs = append(winUUIDs, h.UUID)
+		default:
+			return nil, ctxerr.Errorf(ctx, "unsupported platform %s", h.Platform)
+		}
+		res[h.UUID] = false
+	}
+
+	foo := func(stmtFn func(string) string, uuids []any, mp map[string]bool) error {
```

Comment:
`foo`? Was probably a placeholder waiting for a name?

Reply:
LOL, thank you!